lib/sub_diff.rb

```diff
@@ -1,14 +1,31 @@
 require 'delegate'
 require 'forwardable'
 require 'sub_diff/buildable'
-require 'sub_diff/adapter'
 require 'sub_diff/builder'
 require 'sub_diff/collection'
 require 'sub_diff/diff'
 require 'sub_diff/differ'
+require 'sub_diff/factory'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.